requirements/audio_test.txt

```diff
@@ -6,3 +6,4 @@ mir-eval >=0.6, <=0.7
 speechmetrics @ git+https://github.com/aliutkus/speechmetrics
 fast-bss-eval >=0.1.0, <0.1.5
 torch_complex <=0.4.3  # needed for fast-bss-eval
+srmrpy @ git+https://github.com/jfsantos/SRMRpy
```

Comment:
note this wont be installed with pypi

Reply:
Turned out this repo is not tested, so we shall not rely with parity on their numbers
ref: https://github.com/jfsantos/SRMRpy/pull/15